bpm_projects_api/apis/project.py

```diff
@@ -29,19 +28,29 @@
 })
 
 
+filter_parser = ns.parser()
+filter_parser.add_argument('short_name',
+                           type=query_str(3, 100),
+                           ignore=True,
+                           help='Name or Strign to search')
+filter_parser.add_argument('active', type=inputs.boolean,
+                           help='Is the project active?')
+
+
 @ns.route('/')
 class Projects(Resource):
     @ns.doc('list_projects')
+    @ns.expect(filter_parser)
+    @ns.response(204, 'No match for your search')
     @ns.marshal_list_with(project, code=200)
-    @token_required
     def get(self):
-        """List all projects"""
-        return project_dao.get_all()
+        """List projects whit filters"""
+        filter_data = filter_parser.parse_args()
+        return project_dao.search_project_name_or_state(filter_data)
```

Comment:
This name is fixed to just search project by name or state. We are trying to implement an abstraction of filtering which uses those 2 parameters, but its not fixed to them. I would recommend a more accurate name related to filtering projects (not search by 2 specific parameters which has no relevance at all).

Reply:
do you refer only to the function name?
I think this name is better for the function intention
```suggestion
        return project_dao.search_filtered_projects(filter_data)
```